docker-compose.yml

```diff
@@ -2,26 +2,26 @@
 version: '3'
 services:
   arangodb:
-    image: arangodb:3.5.1
+    image: arangodb:3.7.1
```

Comment:
Use Arangodb 3.7.3 due to arangodb/arangodb#12693

Reply:
We should update this on GitHub pages as well.